features/support/shared_steps.js

```diff
@@ -103,6 +103,7 @@ module.exports = function () {
                             time = hasRoute && json.routes[0].duration,
                             weight = hasRoute && json.routes[0].weight;
 
+                        distance = Math.round(distance);
                         if (headers.has('distance')) {
```

Comment:
ghoshkaj [7:46 PM]
@chaupow distances in in the Route are rounded here https://github.com/Project-OSRM/osrm-backend/blob/faff2c774d09a7227c77ae4fa40d22d54bb00b45/include/engine/guidance/assemble_leg.hpp#L214 and also here: https://github.com/Project-OSRM/osrm-backend/blob/faff2c774d09a7227c77ae4fa40d22d54bb00b45/src/engine/api/json_factory.cpp#L188